pkg/standalone-utils/test/VaultActionsStableJoinExit.test.ts

```diff
@@ -723,6 +724,237 @@ describe('Vault Actions - Stable Pools', () => {
     });
   });
 
+  describe('exit pool in recovery mode', () => {
+    const amountInBPT = fp(1);
+
+    async function getBPT(poolId: string): Promise<TokenList> {
+      return new TokenList([await Token.deployedAt(getPoolAddress(poolId))]);
+    }
+
+    context('when caller is not authorized', () => {
+      it('reverts', async () => {
+        await expect(
+          relayer.connect(other).multicall([
+            await encodeExitPool(vault, relayerLibrary, tokens, {
+              poolKind: PoolKind.COMPOSABLE_STABLE_V2,
+              poolId: poolIdStable,
+              userData: BasePoolEncoder.recoveryModeExit(amountInBPT),
```

Comment:
(unrelated with this PR): the JS pool encoder uses `(uint256, uint256)`, whereas the code in Solidity uses `uint8` for the enum. Should we rectify that in a different PR?

Reply:
I think this is generally inconsistent all over the place; we've noted that before, and it might be a big change to a lot of files.

We may have even had this conversation before, about whether it's worth it, and (I believe) concluded it should be fine to have JS all be uint256, which should still resolve to the same value whether the underlying Solidity is 8, 16, 32, etc. (it can only be equal or smaller), even if the definition of enum changes at some point.

In fact, it might even be "safer" to leave JS at 256, so that we don't have to change it if enums go from uint8 to uint16 in a future version